commands/zip/zip-command-encryption.md

```diff
@@ -0,0 +1,13 @@
+### Zip a directory with encryption
+
+`zip -er {{ZIP_FILE_NAME}} {{DIRECTORY_PATH}}`
+
+- <b>ZIP_FILE_NAME</b>: Filename for new zipped file
+- <b>DIRECTORY_PATH</b>: Directory relative/absolute path, which needs to be zipped
+
+#### Example:
+
+`zip -er ak-cli.zip /ak-cli`
+
+
+<img src="../../gifs/zip-command.gif" alt="Zip Command"/> <br>
```

Comment:
You can remove this line, this gif is related to other zip command

Reply:
How do you create these Gifs?